src/wkt/src/duration.rs

```diff
@@ -164,10 +164,24 @@ impl crate::message::Message for Duration {
     fn typename() -> &'static str {
         "type.googleapis.com/google.protobuf.Duration"
     }
-    fn to_map(&self) -> Result<crate::message::Map, crate::AnyError> {
-        crate::message::to_json_other(self)
+
+    // Override the default serializer to use custom serialization
+    fn serializer() -> Box<dyn crate::message::MessageSerializer<Self>>
+    where
+        Self: Sized,
+    {
+        Box::new(DurationSerializer)
+    }
+}
+
+struct DurationSerializer;
```

Comment:
... then same goes for `Duration`, `Timestamp`, all the types in `rstruct.rs`, etc.

Reply:
Done.